<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Temporizador</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnstart">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnpause">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnreset">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="pantalla">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelHora">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Minutos</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMinuto">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Segundos</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSegundos">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mili</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHora">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMinuto">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSegundo">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
